tests/core/test_signal.py

```diff
@@ -258,14 +259,14 @@ async def test_assert_configuration(mock_readable: DummyReadable):
 async def test_signal_connect_logs(caplog):
     caplog.set_level(logging.DEBUG)
     mock_signal_rw = epics_signal_rw(int, "pva://mock_signal", name="mock_signal")
-    await mock_signal_rw.connect(mock=True)
+    await mock_signal_rw.connect(mock=False)
     assert caplog.text.endswith("Connecting to mock+pva://mock_signal\n")
 
 
 async def test_signal_get_and_set_logging(caplog):
     caplog.set_level(logging.DEBUG)
     mock_signal_rw = epics_signal_rw(int, "pva://mock_signal", name="mock_signal")
-    await mock_signal_rw.connect(mock=True)
+    await mock_signal_rw.connect(mock=False)
```

Comment:
Again, won't this fail? This test needs a mock signal so `mock=True` to mock the real `PvaSignalBackend`.

Reply:
outdated